test/response_test.dart

```diff
@@ -25,6 +26,30 @@ void main() {
     });
   });
 
+  test('supports a Uint8List body without copying', () async {
+    var bytes = Uint8List(10);
+    var response = Response.ok(bytes);
+
+    expect(response.contentLength, 10);
+    expect(identical(await response.read().first, bytes), true);
```

Comment:
```suggestion
    expect(await response.read().single, same(bytes));
```

Using `single` guarantees there aren't extra elements. Using `same` over comparing against the literal `true` will improve error messages if this fails.

Reply:
Ahh, I was looking for that matcher but didn't find it - thank you!